tests/conftest.py

```diff
@@ -0,0 +1,47 @@
+"""Pytest configuration for the sdx package tests."""
+
+import os
+
+from pathlib import Path
+
+import pytest
+
+from dotenv import load_dotenv
+
+
+def pytest_configure(config):
+    """Configure pytest environment."""
+    env_path = Path(__file__).parent / '.env'
+
+    if env_path.exists():
+        load_dotenv(dotenv_path=env_path)
+    else:
```

Comment:
.env.tpl is just a template .. there is no real values there.
So this "else" is not necessary